samples/EventHub/allowed-event-hub-firewall-ip/azurepolicy.json

```diff
@@ -0,0 +1,39 @@
+{
+  "type": "Microsoft.Authorization/policyDefinitions",
+  "name": "allowed-event-hub-firewall",
+  "properties": {
+    "displayName": "Allow IP for event hub firewall",
+    "policyType": "Custom", 
+    "description": "This policy allows only restricted IPs for Event hub firewall",
+    "mode":"All",
+  "parameters": {
```

Comment:
>" [](start = 2, length = 1)

please reformat all of these JSON files so the indentation is correct

Reply:
Done